### PR TITLE
Azure Samples

### PR DESCRIPTION
**Pull request by arturl**
This is one of the samples for my blog post. There will be more in subsequent pull requests.

### CODE REVIEW

File: Azure/WeatherStation/Package.appxmanifest [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+<Package
+  xmlns="http://schemas.microsoft.com/appx/manifest/foundation/windows10"
+  xmlns:mp="http://schemas.microsoft.com/appx/2014/phone/manifest"
+  xmlns:uap="http://schemas.microsoft.com/appx/manifest/uap/windows10"
+  IgnorableNamespaces="uap mp">
+
+  <Identity
+    Name="47d77310-4819-4b33-aabc-2324f2167015"
+    Publisher="CN=arturl"
```
Comment (bfjelds) [inline Azure/WeatherStation/Package.appxmanifest]: make the publisher some variant of msft

File: Azure/WeatherStation/project.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
+﻿{
+  "dependencies": {
+    "Microsoft.ApplicationInsights": "1.0.0",
```
Comment (bfjelds) [inline Azure/WeatherStation/project.json]: i'd suggest removing all the applicationinsights code

File: Azure/WeatherStation/MainPage.xaml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+﻿<Page
```
Comment (bfjelds) [inline Azure/WeatherStation/MainPage.xaml:1]: is there any UI to this?  maybe consider making it a background application?

File: Azure/WeatherStation/Package.appxmanifest (hunk 1 above)
Comment (arturl) [inline Azure/WeatherStation/Package.appxmanifest]: Thanks, I addressed this and other comments as well.

**Comment by arturl** [comment]
Updated!

**Pull request merged**